dev-packages/rollup-utils/bundleHelpers.mjs

```diff
@@ -59,7 +59,7 @@ export function makeBaseBundleConfig(options) {
     plugins: [rrwebBuildPlugin, markAsBrowserBuildPlugin],
   };
 
-  // used by `@sentry/integrations` and `@sentry/wasm` (bundles which need to be combined with a stand-alone SDK bundle)
+  // used by `@sentry/wasm` & pluggable integrations from core/browser (bundles which need to be combined with a stand-alone SDK bundle)
```

Comment:
(not relevant for this PR): This reminds me, we should also move `@sentry/wasm` into browser. It's also just an integration. So now that we can build and publish integration CDN bundles in browser, I think it should be fairly straight forward to bring it in.